chore: user guide (#539)
jst-seminar-rostlab-tum/eutop-mediamind: docs/USER_GUIDE.md

| Access | Description                                                                                                                                                          |
| ------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Owner  | Owner of the profile (only one possible). The owner can read/edit the profile and transfer ownership to another member (maintainer & admin are treated like owners). |
| Write  | Same rights as the owner without the option to transfer ownership.                                                                                                   |
| Read   | They can only see the profile, meaning see which topics are chosen and read the matched articles together with the topics & keywords which caused the match.         |

## Webapp Features

The webapps main purpose is to administrate the preferences of users, organisations & search-profiles.
But additional also results are served to the user and can be looked through. The folllwing views are available.

### Dashboard

### Edit / Create Search Profiles

> ### Search Profile Overview

Reviewer: Maybe we should add in here what the 4 pages of a search profile can do. I am especially thinking of the email lists and how it works with the subscriptions. internal have the orga subscriptions and external the search profile subscriptions... 

It is written on the subscription page of the search profile but maybe we should specify that here.

Author: jep agreed, but that should go into the edit section (line 59) @neher42 this is your page, can you please take care?

Reviewer 2: yes please

Reviewer 2: @neher42 is this done? Have you included topics and general here?

# MediaMind User Guide

MediaMind is an AI-based news analysis tool, which allows users & organisations to select their interests and receive auto-generated reports via email based on their preference (semantic not syntactic search).

This guide will introduce the most important concepts, as well as a rough overview of the features available to the user.

## General

### Terminology

#### Search Profile

> A search profile is a specification of topics and keywords by the user, to receive daily reports about articles, which have been published and are matched to the interests of this search profile.

Reviewer 2: ```suggestion
A search profile is a specification of topics and keywords that define the user’s areas of interest. Based on this specification, MediaMind delivers daily reports containing relevant articles.
```

#### Topics & Keywords

> A topic consists of multiple keywords. For your search profile you can create multiple topics with multiple keywords. MediaMind will semantically match articles to your profile based on the topcis defined by the keywords.

Reviewer 2: ```suggestion
Each search profile consists of one or more topics, and each topic contains multiple keywords. MediaMind uses these keywords to match news articles based on their meaning and context, not just exact word matches.
```

#### Subscriptions

> Sometimes also called Domain. MediaMind crawls articles from different news paper domains. Legally, you are only allowed to receive full texts for domains where your organisation has a valid subscription, although MediaMind might have more subscriptions than you have. (see Organisation Settings)

Author: Cleared it in the frontend

> If you dont have a valid subscription, you will still receive summaries for articles from these domains, if they are matched to your search profile.

Reviewer 2: ```suggestion
If you do not have a valid subscription, you will still receive summaries of articles from these domains, provided they match your search profile. However, access to the full article content will not be available.
```

> Note: For a search profile you can again select Subscriptions. These will then be used for external mail addresses from this search profile. (See search profile edit)

#### Organisations

An organisation is required to use MediaMind. If you want to use MediaMind for your organisation, contact a MediaMind admin, to create an organisation for you.

> If new users should be added to your organisation, they have to first create an account and then, have to be added by a MediaMind admin to your organisation, before they can use the application.

Reviewer 2: ```suggestion
New users must first create an account. After that, a MediaMind administrator must assign them to your organization before they can access the application.
```

(see Admin Settings)

### Authorization Concept

#### User Roles

| Role       | Description                                                                                                                                                                                                                                                                                                          |
| ---------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | Admin      | Application admins of Mediamind. The can create organisations and assign users to organisations. They can create Subscriptions (with username & password) which MediaMind uses to crawl articles. Additionally they can see and edit all search profiles in Mediamind                                                |

Reviewer 2: ```suggestion
| Admin      | Application admins of MediaMind. The can create organisations and assign users to organisations. They can create Subscriptions (with username & password) which MediaMind uses to crawl articles. Additionally they can see and edit all search profiles in MediaMind                                                |
```

| Maintainer | A maintainer is the admin of an organisation, so to speak. The can select organisation preferences (see Organisation Settings) and can edit & see all search profiles of their organisation. Additionally, they can change the owner of a search profile, which can otherwise only be done by the owners themselves. |
| Member     | Normal user within MediaMind which is assigned to an organisation. They can create profiles, edit them, add read/write access for other members & transfer ownership of their own search profiles.                                                                                                                   |

#### Search Profile Access

| Access | Description                                                                                                                                                          |
| ------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | Owner  | Owner of the profile (only one possible). The owner can read/edit the profile and transfer ownership to another member (maintainer & admin are treated like owners). |

Reviewer 2: ```suggestion
| Owner  | Each profile has exactly one ownder. The owner has full access rights and can view and edit the profile. In addition, the owner can transfer ownership to another user. Users with the roles Maintainer or Admin are treated as owners and have the same permissions. |
```

> | Write  | Same rights as the owner without the option to transfer ownership.                                                                                                   |

Reviewer 2: ```suggestion
| Write  | Can do the same as the owner, except transfering ownership.                                                                                                   |
```

| Read   | They can only see the profile, meaning see which topics are chosen and read the matched articles together with the topics & keywords which caused the match.         |

## Webapp Features

The webapps main purpose is to administrate the preferences of users, organisations & search-profiles.
But additional also results are served to the user and can be looked through. The folllwing views are available.

### Dashboard

The dashboard serves as the main entry point into the MediaMind web application. The dashboard contains an overview of all search profiles which the user has access to. The individual search profile cards serve as an entry into the [Search Profile Overview](#search-profile-overview) through the "Explore" button. They also display the users access rights to the profile as well as the visibility of the profile. 

<img src="assets/dashboard.png" alt="Dashboard" width="800">

> #### Edit / Create Search Profiles

Reviewer 2: I feel the title is not well passing.

The contant barely speaks about creating and editing, but rather about the 3 dots and the this one can do to a search profile.

The profile cards provide shortcuts for deleting and editing the respective profile or accessing the [Report Download Center](#download-center). Profile creating is also handle through the dashboard via the "Plus" icon next to the "Profile" header.

<img src="assets/profile-options.png" alt="Profile Options" width="200">

### Search Profile Overview

> This view contains a filter bar and a listing of all articles that were matched with your Search Profile. Each Article has an overall relevance score based on all topics of your profile and an additional matching score for each single topic. With the search bar you can also directly search in the Vector Database for similar articles to your input. The search bar, per default, sorts based on relevance, all sources and topics are included and the timeframe is this day today.

Reviewer 2: ```suggestion
This view contains a filter menu and a listing of all articles that were matched with your Search Profile. Each Article has an overall relevance score based on all topics of your profile and an additional matching score for each single topic. With the search bar you can also directly semantically search the Vector Database for similar articles to your input. The search bar, per default, sorts based on relevance, all sources and topics are included and the timeframe is this day today.
```

<img src="assets/search-profile-overview.png" alt="Search Profile Overview" width="800">

### Article Page

The article page features an integrated in-app reader for articles that match a selected search profile. In addition to the article content, the page presents a quantitative correlation between the topics and keywords of the search profile and those relevant to the specific article. Identified entities such as people and industries are also displayed. Furthermore, metadata including the publication date and original language is provided. The article page is accessed by clicking on the corresponding article in the [Search Profile Overview](#search-profile-overview).

Users can access the original article by clicking the “Original” button located at the top right of the page.

> If a subscription is required to access the full article, an alert is displayed to notify the user. In such cases, only a summary of the article is available.

Reviewer 2: ```suggestion
If a subscription is required to access the full article , but the user's organization does not have one, an alert is displayed to notify the user. In such cases, only a summary of the article is available.
```

![Personal Settings](assets/personal-settings.png)

### Breaking News

The Breaking News view is accessible via the entry at the top of the [Dashboard](#dashboard) page. This section displays significant global events that are currently receiving widespread attention.

### Download Center

The daily reports generated for each [Search Profile](#search-profile) are accessible long term through the download center of that search profile. The download center of a search profile can be accessed either via the "Reports" button displayed on the top right of the [Search Profile Page](#search-profile-overview) or through the [Quick Options](#edit--create-search-profiles) menu of a profile card.

### Admin Settings

This page is only accessable for MediaMind admin. You can create and edit Organizations here and add users with roles to them. You can also create and edit Subscriptions there. The attribute paywalled enables or disables the username and password for a subscription. For safety reasons you don't get your password shown here. If you want to set a new password, you can still do that by just typing it in and confirm.

![Admin Settings](assets/admin-settings.png)

### Crawler Stats

> This page is only accessable for MediaMind admins. You can get an overview of the status of the crawling grouped by domains. Thus, you see, whether crawling was not successfull or where you should have another look.

Reviewer 2: ```suggestion
This page is only accessible to MediaMind administrators. It provides an overview of the crawling status, grouped by domain. This allows you to identify unsuccessful crawls and determine where further investigation may be needed.
```

![Crawler Stats](assets/crawler-stats.png)

### Organisation Settings

> This page is only accessable for organisation maintainers.

Reviewer 2: ```suggestion
This page is only accessible to organisation maintainers.
```

There are two kinds of preferences which should be maintained on an organisation level.

You can decide via a true/false toggle, whether you want to receive the daily report as pdf directly via email or as a signed link (which is valid for a week and can be regenerated in the webapps Download Center) to the pdf.

> The second option is, that you need to select from which subscriptions/domains the full text will be sent to your organisations internal emails.

Reviewer 2: ```suggestion
The second option allows you to specify for which sources your organization holds a valid subscription. The full article content will be included in internal emails and in the webapp for the selected sources.
```

> Legal notice: Although it is possible to select subscriptions even if your organisation does not have a valid subscription/account for this domain, this is legally forbidden.
> Only select domains, where your organisation has a valid subscription!

![Organisation Settings](assets/organisation-settings.png.png)

## ChatBot Feature

The MediaMind Email Chatbot lets you get quick answers by replying to emails you receive from MediaMind.

**How to use:**

1. Reply directly to any email you received from MediaMind (such as your daily news report).
2. Write your question or request in the reply (e.g., "Can you summarize today’s report?" or "Can I get this in another language?").

> 3. Send your reply. You will receive an answer by email, usually within a few minutes.

Reviewer 2: ```suggestion
3. Wait for a response. You will receive an answer by email, usually within a few minutes.
```
